README.md

```diff
@@ -0,0 +1,122 @@
+<a name="readme-top"></a>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
- Great Job for adding the readme template which was suggested by the previous reviewer.
- Kindly All list of content has to be available in the table of content the only optional section is a FAQ.

Reply:
Thank you for your contribution.
Please, I need more clarification.

    Modify this file to match your project and remove sections that don't apply.
    
    REQUIRED SECTIONS:
    - Table of Contents
    - About the Project
      - Built With
      - Live Demo
    - Getting Started
    - Authors
    - Future Features
    - Contributing
    - Show your support
    - Acknowledgements
    - License
    
    OPTIONAL SECTIONS:
    - FAQ

My questions : 
- "and remove sections that don't apply " What does it mean?
- Do I need to have all the sub-sections of the required sections?
- For example: Am I required to have the "Install" sub-section of the "Getting Started" section?

Reply:
@binhussen 
Thank you for your contribution.
Please, I need more clarification.

    Modify this file to match your project and remove sections that don't apply.
    
    REQUIRED SECTIONS:
    - Table of Contents
    - About the Project
      - Built With
      - Live Demo
    - Getting Started
    - Authors
    - Future Features
    - Contributing
    - Show your support
    - Acknowledgements
    - License
    
    OPTIONAL SECTIONS:
    - FAQ

My questions :

- "and remove sections that don't apply " What does it mean?
- Do I need to have all the sub-sections of the required sections?
- For example: Am I required to have the "Install" sub-section of the "Getting Started" section?

Reply:
Hello @Matthieu96Code , yes! do include them. this Is your first project, and the aim is to get you to be familiar will using all resources for future complex projects. 
- [ ] Getting started section is important, as it gives your viewers a guide on how to use your project.
- [ ] For installation, still include it and say something like "This app does not need special installment". Always include all sections, because going forward all of them will have an entry. i hope am clear enough 😃